### PR TITLE
Use network settings specified in the config

### PR DESCRIPTION
**Pull request by vrde**
Problem: the parameter `rootNetwork` contained in the configuration is
ignored, while `rootNetworkId` is used to match the `rootNetwork` from
`./src/utils/networks.js`. This prevents a node operator to connect to
the endpoint they want.

Solution: use `rootNetwork` and remove `./src/utils/networks.js`.

Fix: #347 
Fix: #352

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,19 +59,21 @@ Your local port `9999` forwards now to the remote host on address `127.0.0.1` an
...
   "network": "leap-testnet-gamma",
   "networkId": 1341,
+```
 
 - `bridgeAddr` — leap [Bridge](https://github.com/leapdao/leap-contracts) contract address
 - `operatorAddr` — leap [Operator](https://github.com/leapdao/leap-contracts) contract address
 - `exitHandlerAddr` — leap [ExitHandler](https://github.com/leapdao/leap-contracts) contract address
-- `rootNetwork` — *DEPRECATED. Use `rootNetworkId` instead.* Ethereum provider url (e.g. `https://rinkeby.infura.io`)
-- `rootNetworkId` — root network id (see [src/utils/networks.js](https://github.com/leapdao/leap-node/blob/master/src/utils/networks.js),
+- `rootNetwork` — Ethereum provider URL (e.g. `https://rinkeby.unfura.io`)
```
Comment (troggy) [inline README.md]: 😂

Comment (vrde) [reply README.md]: Oh I think I mistyped it for real :grimacing:

**Comment by vrde** [comment]
> I think, provider URL should be specified via CLI args. In my perspective, the config file contains the configuration for the network, the conditions. All the particularities (like provider URL or ports to use) are to be specified via CLI args.

Would this be a breaking change? I think it's correct to have the root network in the config. As an example, think about a sysadmin/devops that has to setup a server running `leap-node`. All configuration files are usually in the `/etc` dir, and they would expect those parameters to be in there. If we make the CLI args mandatory, they would have to open the service file for `systemd` and change _part of the configuration_ from there.

**Comment by troggy** [comment]
@vrde there are failing unit tests

**Comment by troggy** [comment]
if there is no `rootNetwork` in the config, the node throws not very informative

> unhandledRejection Error: Provider not set or invalid

I suggest to show the message like "Please set the rootNetwork in the config to the provider for chainId XXXX"

**Comment by vrde** [comment]
> @vrde there are failing unit tests

Thanks, I forgot to run them on my last commit. I had to simplify the mock implementation, it was making too many assumptions.

> I suggest to show the message like "Please set the rootNetwork in the config to the provider for chainId XXXX"

Good point, it's done.

**Comment by troggy** [comment]
integration tests are broken 👀

**Pull request merged**